src/app/pages/services/components/service-rsync/rsync-configuration/configure-rsync/configure-rsync.component.ts

```diff
@@ -1,45 +1,83 @@
-import { Component } from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef, Component, OnInit,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
 import helptext from 'app/helptext/services/components/service-rsync';
-import { FormConfiguration } from 'app/interfaces/entity-form.interface';
 import { RsyncConfigUpdate } from 'app/interfaces/rsync-config.interface';
-import { EntityFormComponent } from 'app/pages/common/entity/entity-form/entity-form.component';
-import { FieldConfig } from 'app/pages/common/entity/entity-form/models/field-config.interface';
-import { WebSocketService } from 'app/services/ws.service';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { WebSocketService, DialogService, AppLoaderService } from 'app/services';
 
+@UntilDestroy()
 @Component({
   selector: 'app-configure-rsync',
-  template: '<entity-form [conf]="this"></entity-form>',
+  templateUrl: './configure-rsync.component.html',
+  styleUrls: ['./configure-rsync.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CconfigureRYSNCComponent implements FormConfiguration {
-  queryCall = 'rsyncd.config' as const;
-  title = helptext.configureFormTitle;
-
-  route_success: string[] = ['services'];
-
-  fieldConfig: FieldConfig[] = [
-    {
-      type: 'input',
-      name: 'port',
-      placeholder: helptext.rsyncd_port_placeholder,
-      tooltip: helptext.rsyncd_port_tooltip,
-      value: helptext.rsyncd_port_value,
-    },
-    {
-      type: 'textarea',
-      name: 'auxiliary',
-      placeholder: helptext.rsyncd_auxiliary_placeholder,
-      tooltip: helptext.rsyncd_auxiliary_tooltip,
-    },
-  ];
+export class CconfigureRYSNCComponent implements OnInit {
```

Comment:
Funny name :) Seems like it needs renaming to follow the naming convention. Also, we have `RYSNCConfigurationFormComponent` that makes me confused what is the difference between these components. It would be nice if we can distinguish them based on names.